inc/datahandlers/user.php

```diff
@@ -1783,4 +1783,61 @@ function clear_profile($delete_uids=false, $gid=0)
 			remove_avatars($uid);
 		}
 	}
+
+	public function verify_signature()
+	{
+		global $mybb, $parser;
+
+		$parser_options = array(
+			'allow_html' => $mybb->settings['sightml'],
+			'filter_badwords' => 1,
+			'allow_mycode' => $mybb->settings['sigmycode'],
+			'allow_smilies' => $mybb->settings['sigsmilies'],
+			'allow_imgcode' => $mybb->settings['sigimgcode'],
+			"filter_badwords" => 1
+		);
+
+		if($this->data['showimages'] != 1 && $this->data['uid'] != 0)
```

Comment:
If you read my comment again, I said it was entirely wrong from the beginning... Someone copy pasted it earlier, probably when the setting was added - it shouldn't have been there in the first place.